styles.css

```diff
@@ -2,6 +2,36 @@ p {
   font-family: Roboto;
   font-weight: 500;
 }
+body {
+  margin: 0;
+}
 a {
   font-family: Roboto;
 }
+.wrapper {
+ margin: auto;
+ width: 80%;
+ }
+.logo, .menu {
+ display: inline-block;
+}
+.logo {
+ margin-right: 63%;
+}
+.header {
+ display: flex;
+ justify-content: space-between;
+}
+.logo {
+ padding: 40px 0 40px 0;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/IllenzeerAnna28/pullRequest?prid=6364836)